test/blackbox-tests/test-cases/actions/concurrent-multi-diff.t/run.t

```diff
@@ -0,0 +1,84 @@
+The use of (concurrent ) illustrated in the context of diffing multiple files.
```

Comment:
```suggestion
The use of `(concurrent )` illustrated in the context of diffing multiple files.
```

Reply:
This is a cram test and will not render monospace so we don't format it.